File: Python Exams/01. Basics Exam Collection/Basics Online Exam - 6 and 7 April 2019/02. Godzilla vs. Kong/02. Godzilla vs. Kong.py
budget, statist, clothes = [float(input()) for _ in range(3)]

decor_sum = budget * 0.10
if statist > 150:
    clothes_sum = int(statist) * clothes  * 0.90
else:
    clothes_sum = int(statist) * clothes

subtotal_sum = decor_sum + clothes_sum

if budget >= subtotal_sum:
    print(f'Action!\nWingard starts filming with {budget - subtotal_sum:.2f} leva left.')
else:
    print(f'Not enough money!\nWingard needs {abs(budget - subtotal_sum):.2f} leva more.')


'''
9587.88
222
55.68
'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 6 and 7 April 2019/02. Godzilla vs. Kong/02. Godzilla vs. Kong.js
function godzillaVsKong(...arg) {
    let [budget, statist, clothes] = [...arg]

    let decorSum = budget * 0.10
    if (statist > 150) {
        clothesSum = statist * clothes * 0.90
    } else {
        clothesSum = statist * clothes
    }

    let subtotalSum = decorSum + clothesSum

    if (budget >= subtotalSum)
        return `Action!\nWingard starts filming with ${(budget - subtotalSum).toFixed(2)} leva left.`
    return `Not enough money!\nWingard needs ${(subtotalSum - budget).toFixed(2)} leva more.`
}

//console.log(godzillaVsKong(20000, 120, 55.5));